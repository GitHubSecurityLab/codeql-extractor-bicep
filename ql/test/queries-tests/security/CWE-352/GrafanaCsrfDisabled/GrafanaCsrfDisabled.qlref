security/CWE-352/GrafanaCsrfDisabled.ql
