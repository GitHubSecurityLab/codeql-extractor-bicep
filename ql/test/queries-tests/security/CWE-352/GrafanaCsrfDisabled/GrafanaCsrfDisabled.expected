| app.bicep:12:26:12:30 | false | CSRF protection (csrfAlwaysCheck) is explicitly disabled in Grafana configuration, which can lead to cross-site request forgery attacks. |
